docs/course-information.md

```diff
@@ -7,43 +7,51 @@ description: >-
+Software is an integral part of nowadays research. A [UK survey in 2014](https://zenodo.org/record/1183562#.YTHZ91uxUUE) found that 7 out of 10 researchers could not conduct research without software. However, research software engineering does not yet get the necessary attention in research and in teaching. All this applies to simulation software in particular.
+
+Read more:
+
+- Association of German Research Software Engineers: [de-RSE](https://de-rse.org/en/)
+- [Position paper of de-RSE](https://f1000research.com/articles/9-295/v2) 
+
+
 
 ## Content of the course
 
-The course is about simulation software. More precisely, we discuss the software tools used to ensure good software engineering for open-source simulation tool such as preCICE, DuMUX, FEniCS and so on. This is not a course focused on programming and programming paradigms, but on the techniques and the correspnding tools. Some examples for course contens are version control (Git, GitHub, GitLab), virtualization/containerization (Docker), Testing (GitHub, GitLab, Containers), Documentation  (Jekyll, mkdocs, sphinx).
+The course is about simulation software. More precisely, we discuss the software tools used to ensure good software engineering for open-source simulation software such as preCICE, DuMUX, TRILINOS, or FEniCS. This is not a course focused on programming and programming paradigms, but on the techniques and the correspnding tools. Some examples are version control (Git, GitHub, GitLab), virtualization/containerization (Docker), continuous integration (GitHub Actions, GitLab CI), or documentation  (Jekyll, mkdocs, sphinx).
 
 ## Target audience
 
-This course is aimed at master-level students from Computer Science, Simulation Sciences and related fields, but is also open to PhD students or anyone else interested. We expect you to have some knowledge in software development, basic tools and programming. However, we will also give you extra information or point to additional resources if you have the feeling you need to catch up with some topic.
+This course is aimed at master-level students from Computer Science, Simulation Sciences and related fields, but is also open to PhD students or anyone else interested. We expect you to have some knowledge in software development, basic tools and programming. However, we also give you extra information or point to additional resources if you have the feeling you need to catch up with some topic.
```

Comment:
Should we change "Simulation Science" to "Simulation Technology"? This would be close to the phrasing in Stuttgart